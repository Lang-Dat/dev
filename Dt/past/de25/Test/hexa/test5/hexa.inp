01010000111100101101000101111100011101110001101
00010
10010110011111101000010
00101101110001001001011101110101001110011010001001011101010001100000111001011110011001
01001111010100011101101010011011000011010101000001100001101011100100101100011010
0101100011111000010000100001101100101010001001100000010110
11010011001110100110001111101111101010001001011110011011101010111000
1001101110001100001111111000100100110100
10001001011111101010
0111111011101010010101010010001110101011110111100110000000
011
00000000111001111011000000100101100101000001110001000111110001001001000011111010010
0111011101001011101001101111000100100010110011100110010000101100111110011
00101011010111111010000101111111100010110011101000010100011101000010011000011101011111101101000
011011100100011000001001101000001001010010111000
0011011000010001010100011000101000100110100100011100010111110100110100111010010101110010000000101111
01100111010110101011001011100110011100101110010001010010001111100
1100011100000001010000000010001010110101000000000101
0000001100001000110000001111011110011010111011111010000100111011
00111110110101010101111110011110
01101011010010110000000110110100111011010001000101010
01101010001111111011
1000000101000111
0000100100001001001011001001101011000000100010000001101110100
010101001101110010001010111111000010100001110110010010010110010
0101111011101000111010
1010100110111010000000100010010010000001101101100110100100011011110001010000000
011010110011011
1000011110101101000
1101111011011100001100110100101010111000101110110110110100110001101101001000101111100
111010111101000011000111010110101110001101000111000011111010111100110
101011100110010010010011
01110
0100001100000100110101010110011001010001011000110101001111110000110011010111
0010110011001110100001100
01100010010101110110000100111001011011010100010001000000110010100100011001010010110000101000
10001101011011111001111010011011110001000110101010001111100001100010010101111
00101110000100110011010111010001000100000001101000101010000101110001000110001101101101010110001
11011010011001001111110010101110100010111111111001011010001000100111010111111
1001100110011101111100001100010111101001100011100010110111000110011111000110101011000101000111
10101100010100001111100101010111000001011101100011001110110010011100
0111100000110101010111011110101100011100100100100101011010110001100101011010110001010000
111
1100000011001011
0010111111000110000110110000110111001101010111100011101110010110100100111110
1101101101000011100111100011000000001010
110100110011111001100110101110001000101010101101111101100100
010011101101001110111111000000000010000100011110001101000001111000101010100100111100111111000110000
01001001111010000001010000010111101110100
11000101010101100001011000001111001111101011001011001011000101101000000001100101010001
