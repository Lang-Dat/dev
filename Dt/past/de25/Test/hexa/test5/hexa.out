287968BE3B8D
2
4B3F42
B7125DD4E689751839799
4F51DA9B0D5061AE4B1A
163E1086CA89816
D33A63EFA8979BAB8
9B8C3F8934
897EA
1FBA9548EAF7980
3
73D812CA0E23E2487D2
EE974DE2459CC859F3
15AFD0BFC59D0A3A130EBF68
6E4609A094B8
3611518A2691C5F4D3A57202F
CEB565CCE5C8A47C
C7014022B5005
308C0F79AEFA13B
3ED55F9E
D6960369DA22A
6A3FB
8147
121259358110374
2A6E457E143B24B2
17BA3A
54DD011240DB348DE280
359B
43D68
1BDB866957176DA636917C
1D7A18EB5C68E1F5E6
AE6493
E
4304D566516353F0CD7
599D0C
625761396D4440CA4652C28
11ADF3D3788D51F0C4AF
17099AE8880D150B88C6DAB1
1B4C9F95D17FCB444EBF
26677C317A638B719F1AB147
AC50F95705D8CEC9C
78355DEB1C9256B195AC50
7
C0CB
2FC61B0DCD5E3B9693E
DB439E300A
D33E66B88AADF64
2769DF80108F1A0F1549E7E30
93D0282F74
31558583CFACB2C5A01951
